CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.3.1 (2018-11-29)
+
+- Fixed select tags to display invalid states on touched [#38](https://github.com/blackbaud/skyux-theme/pull/38)
+- Fixed selected tab mixin to have correct hover color when used with normal tab mixin [#41](https://github.com/blackbaud/skyux-theme/pull/41)
```

Comment:
"Fixed `sky-btn-tab` SCSS mixin to properly style a tab that is both selected and disabled."

Reply:
Wouldn't it be

""Fixed sky-btn-tab SCSS mixin to properly style a tab that is both selected and hovered."?

Reply:
I've updated the text to this. Let me know if that isn't what you want

Reply:
Hm, I saw the `:not(.sky-btn-tab-disabled)` in the mixin. How about...
"Fixed `sky-btn-tab` SCSS mixin to properly style the hover state for a selected, non-disabled tab."
(Make sure to remember the period and the backticks for the mixin name.)

Reply:
Done